recipes/pyinquirer/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright 2018 Oyetoke Toby and contributors
```

Comment:
It looks like the license file is already included in the source distribution, so you should remove this manually packaged copy. Note the in the source distribution it's just called `LICENSE`, you will need to change the `license_file` line in the `meta.yaml`.